www/kubl05/homework-05/main.js

```diff
@@ -0,0 +1,174 @@
+ *     z C na F: vynásobit devíti, vydělit pěti a přičíst 32. 
+ *     z F na C: odečíst 32, vynásobit pěti a vydělit devítkou. 
+ */
+// SPOILER ALERT!
+
+var celsius = 20, fahrenheiht = 100;
+var fahrenheihtTransferred, celsiusTransferred;
+
+fahrenheihtTransferred = ((celsius * 9) / 5) + 32;
+celsiusTransferred = ((fahrenheiht - 32) * 5) / 9;
+
+console.log(celsius + "°C = " + fahrenheihtTransferred + "°F");
+console.log(fahrenheiht + "°F = " + celsiusTransferred + "°C");
+
+/**
+ * Funkce function fonction funktio. Vemte předchozí úlohy a udělejte z nich funkce. Tj. vytvořte funkce, 
+ * které přijímají argumenty, a na základě argumentů po zavolání vypíše výsledek na konzoli. 
+ * Párkrát zavolejte tyto funkce s různými argumenty. V konzoli také vyzkoušejte, zda fungují vaše funkce. 
+ */
+// SPOILER ALERT!
+
+var getAge = function (birthYear) {
+    var currentYear = new Date().getFullYear();
+    var age = currentYear - birthYear;
+
+    return console.log("Pepeho věk je: " + age);
+};
+
+var getFahrenheiht = function (celsius) {
+    var fahrenheihtTransferred = ((celsius * 9) / 5) + 32;
+
+    return console.log(celsius + "°C = " + fahrenheihtTransferred + "°F");
+};
+
+var getCelsius = function (fahrenheiht) {
+    var celsiusTransferred = ((fahrenheiht - 32) * 5) / 9;
+
+    return console.log(fahrenheiht + "°F = " + celsiusTransferred + "°C");
+};
+
+/**
+ * %CENSORED%. Vytvořte funkci, která vezme 2 číselné argumenty a vrátí podíl prvního čísla a druhého čísla 
+ * v procentech. Výsledek vypište do konzole, např. 21 je 50% z 42. Pro zkrácení / zaokrouhlování desetinných 
+ * míst použijte funkci .toFixed(n). Např. var pi = 3.1415926535; pi.toFixed(2); Pozor na dělení nulou! 
+ */
+// SPOILER ALERT!
+
+var getPercent = function (firstNumber, secondNumber) {
+
+    if (secondNumber != 0) {
+        var percent = ((firstNumber / secondNumber) * 100).toFixed(0);
+
+        return console.log(firstNumber + " je " + percent + "%" + " z " + secondNumber);
+    }
+
+    else {
+        return console.log("nulou se nedělí :/");
+    };
+};
+
+/**
+ * Kdo s koho. Vytvořte funkci, která vezme 2 číselné argumenty a vrátí ten větší z nich. Pokud se čísla 
+ * rovnají, vypište, že se rovnají. Vyzkoušejte funkčnost pro celá čísla, desetinná čísla, zlomky. Zkuste 
+ * je párkrát zavolat v kódu a výsledky uložit do proměnných. 
+ */
+// SPOILER ALERT!
+
+var whoIsBigger = function (a, b) {
+    if (a > b) {
+        return a;
+    }
+    else if (a < b) {
+        return b;
+    }
+    else {
+        return a + " = " + b;
+    }
+};
+
+var x, y, z;
+
+x = whoIsBigger(5, 10);
+y = whoIsBigger(5.2, 5.1);
+z = whoIsBigger(5 / 5, 10 / 2);
+
+/**
+ * I can cleary see the pattern. Vytvořte funkci, která vypíše popořadě všechny násobky 13, které jsou menší 
+ * nebo rovno 730. Použijte for loop. 
+ */
+// SPOILER ALERT!
+
+var getMultiplesOfThirteen = function () {
+    var multiples;
```

Comment:
`var multiples = "";` jinak je to undefined, a vysledek by byl `undefined13 26 39`